src/main/kotlin/no/nav/familie/ba/sak/kjerne/vedtak/vedtaksperiode/VedtaksperiodeUtil.kt

```diff
@@ -55,6 +67,43 @@ fun hentPerioderMedUtbetaling(
         )
     }
 
+fun hentPerioderMedUtbetaling(
+    andelerTilkjentYtelse: List<AndelTilkjentYtelse>,
+    vedtak: Vedtak,
+    forskjøvetVilkårResultatTidslinjeMap: Map<Aktør, Tidslinje<Iterable<VilkårResultat>, Måned>>
+): List<VedtaksperiodeMedBegrunnelser> {
+    val utdypendeVilkårsvurderingTidslinje =
+        forskjøvetVilkårResultatTidslinjeMap
+            .tilUtdypendeVilkårsvurderingTidslinjeMap().values
+            .kombinerUtenNull { it.flatten().toSet() }
+            .filtrer { it != null && it.isNotEmpty() }
```

Comment:
IntelliJ driver å klager på denne, det er foreslått disse endringene:

```suggestion
            .filtrer { !it.isNullOrEmpty() }
```

Reply:
fikset i https://github.com/navikt/familie-ba-sak/pull/2765/commits/dcc97ea590fffa4a060a152b1acdcb4c13f62258